clang/test/CodeGen/PowerPC/ppc-full-reg-names.c

```diff
@@ -0,0 +1,78 @@
+// REQUIRES: powerpc-registered-target
```

Comment:
Tests at the wrong layer: https://maskray.me/blog/2021-08-08-toolchain-testing#the-test-checks-at-the-wrong-layer

This should just test that clang passes -target-feature. It's unnecessary to enumerate every combination.

Also, use `--target=` for new tests. `-target ` has been deprecated since 3.4.

Reply:
Yeah, a full on integration test for this is a bit much.  The backend tests LGTM; just making sure the frontend doesn't error when presented with this flag (and the `-no` variant) would be sufficient IMO.